msi/createmsi.py

```diff
@@ -141,6 +143,12 @@ def generate_files(self):
             'Id': 'INSTALLDIR',
             'Name': 'Meson',
         })
+        ET.SubElement(installdir, 'Merge', {
+            'Id': 'VCRedist',
+            'SourceFile': self.redist_path,
+            'DiskId': '1',
+            'Language': '0',
+            })
```

Comment:
__[flake8]__
* [E123] closing bracket does not match indentation of opening bracket's line

<!-- Comment by [SideCI](https://sideci.com) -->